src/components/molecules/TeamCard/index.jsx

```diff
@@ -0,0 +1,18 @@
+import Heading1 from '../../shared/Typography/Heading1';
+import { MeetTheTeam, Heading } from '../../atoms/TeamCard/styles';
+import Heading4 from '../../shared/Typography/Heading4';
+import TeamCard from '../../atoms/TeamCard';
+
+const TeamSection = () => {
+  return (
+    <MeetTheTeam>
+      <Heading>
```

Comment:
Change this tag to something meaningful. _Heading_ may get conflicts in import later.

Reply:
Done